.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Of course don't know how it came here I always add changes via git add -p. Nvm I removed it. Thanks.